data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/lost_empires_of_faerun/le_deities.lst

```diff
@@ -0,0 +1,15 @@
+
+
+Amaunator		DESC:Amaunator (dead)		ALIGN:LN	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Mace (Light)		DOMAINS:Law,Nobility,Planning,Sun,Time
+Auppenser		DESC:Auppenser (dead)		ALIGN:TN	TYPE:Dead		PANTHEON:Psionic		DEITYWEAP:Kukri			DOMAINS:Balance,Mentalism,Protection,Strength
+Bhaal			DESC:Bhaal (dead)		ALIGN:LE	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Dagger		DOMAINS:Death,Destruction,Evil,Retribution
+Gilgeam			DESC:Gilgeam (dead)		ALIGN:LE	TYPE:Dead		PANTHEON:Untheric		DEITYWEAP:Mace (Heavy)		DOMAINS:Air,Strength,Tyranny,War
+Ibrandul		DESC:Ibrandul (dead)		ALIGN:CN	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Greatclub		DOMAINS:Cavern,Darkness,Scalykind,Travel
+Leira			DESC:Leira (dead)		ALIGN:CN	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Dagger		DOMAINS:Charm,Illusion,Spell,Trickery
+Moander			DESC:Moander (dead)		ALIGN:CE	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Mace (Heavy)		DOMAINS:Corruption,Destruction,Evil,Slime
+Myrkul			DESC:Myrkul (dead)		ALIGN:NE	TYPE:Dead		PANTHEON:Human			DEITYWEAP:Scythe		DOMAINS:Death,Evil,Suffering,Undeath
+Ramman			DESC:Ramman (dead)		ALIGN:LN	TYPE:Dead		PANTHEON:Untheric		DEITYWEAP:Khopesh		DOMAINS:Law,Planning,Protection,War
+Tchazzar		DESC:Tchazzar (dead)		ALIGN:CE	TYPE:Dead		PANTHEON:Humans, Dragons	DEITYWEAP:Longsword		DOMAINS:Scalykind,Strength,War
```

Comment:
Please update the PANTHEON entry to use the new FACTSET system - FACTSET:Pantheon|x